kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/challenge3/orders/OrderRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.kodilla.good.patterns.challenges.challenge3.orders;
+
+public class OrderRepository {
+
+    public void saveOrderInArchive(OrderRequest order) {
+        System.out.println("Save order in archive: " + order.getSupplierName() + " , product: " + order.getProduct() + " /Date: " + order.getDateOfOrder());
+    }
```

Comment:
sformatuj date, tak aby byla bardziej user-friendly
 order.getDateOfOrder()
np.
https://www.mkyong.com/java8/java-8-how-to-format-localdatetime/